client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "vue-cli-service serve --mode=development",
```

Comment:
what does this do?

Reply:
[Despite the docs](https://cli.vuejs.org/guide/mode-and-env.html#modes), `serve` here was using `.env.production`. I had to add this to force it to use my `.env` file. I'd describe that as a comment but json doesn't support comments.

Reply:
Interesting, thanks!